core/commands/bitswap.go

```diff
@@ -16,7 +16,7 @@ import (
 
 var BitswapCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "A set of commands to manipulate the bitswap agent",
+		Tagline:          "A set of commands to manipulate the bitswap agent.",
```

Comment:
Cool. I say we go for it, because we haven't largely out of laziness. #2231 looks good.